src/ios/MalwarelyticsPlugin.swift

```diff
@@ -63,16 +67,20 @@ class MalwarelyticsPlugin: CDVPlugin {
     @objc(setClientId:)
     func setClientId(command: CDVInvokedUrlCommand) {
         guardInstance(command) {
-            $0.clientIdentification.userId = command.arguments[0] as? String
-            self.success(command)
+            if let online = $0.online {
+                online.clientIdentification.userId = command.arguments[0] as? String
+                self.success(command)
```

Comment:
Doesn't this mean that when somebody calls this in the "offline mode", the await in js will never return?